pages/book/cells.mdx

```diff
@@ -47,11 +47,34 @@ Every cell, being a [quadtree][quadtree], has an attribute called _level_, which
 
 [Exotic](#cells-kinds) cells have different rules for determining their level, which are described [on this page in TON Docs](https://docs.ton.org/develop/data-formats/exotic-cells).
 
-### Standard representation [#cells-representation]
+### Serialization [#cells-serialization]
+
+Before a cell can be transferred over the network or stored on disk, it must be serialized. There are several common formats, such as [standard `Cell{:tact}` representation](#cells-representation) and [BoC](#cells-boc).
+
+#### Standard representation [#cells-representation]
+
+Standard [`Cell{:tact}`](#cells) representation is a common serialization format for cells first described in the [tvm.pdf](https://docs.ton.org/tvm.pdf). Its algorithm representing cells in octet (byte) sequences begins with serializing the first $2$ bytes called descriptors:
+
+* _Refs descriptor_ is calculated according to this formula: $r + 8 * k + 32 * l$, where $r$ is the number of references contained in the cell (between $0$ and $4$), $k$ is a flag for the cell kind ($0$ for [ordinary](#cells-kinds) and $1$ for [exotic](#cells-kinds)), and $l$ is the [level](#cells-levels) of the cell (between $0$ and $3$).
+* _Bits descriptor_ is calculated according to this formula $\lfloor\frac{b}{8}\rfloor + \lceil\frac{b}{8}\rceil$, where $b$ is the number of bits in the cell (between $0$ and $1023$).
+
+Then, the data bits of the cell themselves are serialized as $\lceil\frac{b}{8}\rceil$ $8$-bit octets (bytes). If $b$ is not a multiple of eight, a binary $1$ and up to six binary $0$s are appended to the data bits.
+
+Next, the $2$ bytes store the depth of the refs, i.e. their [levels](#cells-levels).
```

Comment:
Cell depth is not the same as cell level.
Depth is simply how deep the cell tree is, and level is a completely different thing - you can read about it in tvm.pdf 3.1.6. Also cell levels aren't usually being used apart from working with exotic cells (and even in that case, the whole *usage* is done by a library or TVM itself.

Reply:
Whoops, confused those terms